phpBB/phpbb/textformatter/s9e/renderer.php

```diff
@@ -0,0 +1,256 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+namespace phpbb\textformatter\s9e;
+
+/**
+* s9e\TextFormatter\Renderer adapter
+* @package phpBB3
+*/
+class renderer extends \phpbb\textformatter\renderer
+{
+	/**
+	* @var s9e\TextFormatter\Plugins\Censor\Helper
+	*/
+	protected $censor;
+
+	/**
+	* @var s9e\TextFormatter\Renderer
+	*/
+	protected $renderer;
+
+	/**
+	* @var bool Status of the viewcensors option
+	*/
+	protected $viewcensors = false;
+
+	/**
+	* @var bool Status of the viewflash option
+	*/
+	protected $viewflash = false;
+
+	/**
+	* @var bool Status of the viewimg option
+	*/
+	protected $viewimg = false;
+
+	/**
+	* @var bool Status of the viewsmilies option
+	*/
+	protected $viewsmilies = false;
+
+	/**
+	* Constructor
+	*
+	* @param  \phpbb\cache\driver\driver_interface $cache
+	* @param  string $cache_dir Path to the cache dir
+	* @param  string $key Cache key
+	* @param  factory $factory
+	* @return null
+	*/
+	public function __construct(\phpbb\cache\driver\driver_interface $cache, $cache_dir, $key, factory $factory)
+	{
+		$renderer_data = $cache->get($key);
```

Comment:
in this case I think it's better to directly dump a php class (have a look at https://github.com/symfony/DependencyInjection/blob/master/Dumper/PhpDumper.php https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Routing/Matcher/Dumper/PhpMatcherDumper.php and https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Routing/Generator/Dumper/PhpGeneratorDumper.php)

Also you could save you some work by using \Symfony\Component\Config\ConfigCache (you can see an example in \phpbb\di\container_builder and \phpbb\routing\router)